### DISCUSSION TITLE
See Traefik logs or check if traffic is correctly forwarded to api-gateway

### DISCUSSION THREAD

**Discussion by theliuk** [mia-platform/community#79] [Q&A]
Hi,
we have a problem with a touchpoint which is exposed on the Internet by Traefik.

We don't know why the `api-gateway` is not receiving any traffic.

Is there a way to see Traefik logs to debug?

Thanks,

Luca

**Comment by davidebianchi** [comment]
Hi @theliuk! Are you in dedicated or in shared environment?

**Reply by theliuk** [reply]
Dedicate environment, private cluster.

**Reply by davidebianchi** [reply]
Ok, so you should have access to logs from grafana. Do you see mia-platform namespace?
